Capitalize “Slack” (#1994)
writethedocs/www: docs/conf/portland/2020/writing-day.rst

**How would you customize a style guide?**

We have a draft `Style guide template <https://docs.google.com/document/d/1HxtaiayAJZvF0ZfNjLvRH3vYMvGTEki_TK8hFilQNJ0>`_ and would love feedback from tech writers. (Please use comments rather than track changes in the doc - it is easier to maintain.)

Reach out to @alyssa on WTD Slack if interested.

**Transform templates to your wiki format**

We want to be able to convert our markdown templates to your format? What tools, configuration settings, and how-tos should we have for this?

Reach out to @ScriptAutomate on WTD Slack if interested.

**What should a glossary or word list look like?**

We are kicking off a pilot to define best practices for `cross-domain management of glossaries <https://docs.google.com/document/d/1Fjrl34ErnYammel9WmvXJ3rMWFANjoSiiGyyNSYOXUg/>`_. We want help defining what an ideal glossary entry or preferred word list entry should look like. Work will happen in this `glossary entry <https://docs.google.com/document/d/1wsSLQ_T8skVdlvjF5ayZa5IhKbTazdqm97lHX0qc16Q>`_ page.

Reach out to @alyssa and @cameronshorter on WTD Slack if interested. (@cameronshorter is in the Australian timezone.)

**General chit-chat**

> We hang out in [a group Slack workspace](https://join.slack.com/t/thegooddocs/shared_invite/enQtODkyNjI5MDc0NjE0LTUyNGFiZmU1MjIzNDMwN2E3NmQwODQwZmRkYWI5MDhlMzdjYzg4Nzg4YjM3ODA0NGE4MTgyYzdkMGViMTI2MDM).

Reviewer: ```suggestion
We hang out in [The Good Docs Slack workspace](https://join.slack.com/t/thegooddocs/shared_invite/enQtODkyNjI5MDc0NjE0LTUyNGFiZmU1MjIzNDMwN2E3NmQwODQwZmRkYWI5MDhlMzdjYzg4Nzg4YjM3ODA0NGE4MTgyYzdkMGViMTI2MDM).
```
